core/data/src/main/java/uk/gov/gchq/gaffer/data/element/function/TypeSubTypeValueTuple.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.data.element.function;
+
+import uk.gov.gchq.gaffer.types.TypeSubTypeValue;
+import uk.gov.gchq.koryphe.Since;
+import uk.gov.gchq.koryphe.Summary;
+import uk.gov.gchq.koryphe.tuple.Tuple;
+
+import java.util.Arrays;
+
+import static java.util.Objects.isNull;
+@Since("1.17.0")
+@Summary("Tuple object for TypeSubTypeValue")
+public class TypeSubTypeValueTuple implements Tuple<String> {
+    private TypeSubTypeValue tsv;
+
+    public TypeSubTypeValueTuple() {
+        this(null);
```

Comment:
Might be safer to remove this telescoping constructor behaviour.

Reply:
Why? the issue with telescoping contractors behaviour is the parameter list grows and grows, this has 1 thing in it.